Views/home.stencil

```diff
@@ -3,15 +3,24 @@
 {% block title %}Hackers at Cambridge{% endblock %}
 
 {% block content %}
-  <header class="LandingIntroduction">
-    <h1>Hackers at Cambridge</h1>
-    <p>Cambridge's student tech society</p>
-    <p>Welcome to our beautiful new website</p>
-  </header>
-  <section class="LandingFeature">
-    <h2>Featured</h2>
-    <p>Hack Cambridge</p>
-  </section>
+  <div class="LandingTop">
+    <header class="LandingIntroduction LandingTop__Introduction">
+      <div class="LandingIntroduction__titleContainer">
+        <img src="static/images/hac-logo-dark.svg" alt="Hackers at Cambridge" class="LandingIntroduction__titleImage">
+        <h4 class="LandingIntroduction__tagLine">Cambridge's student tech society</h4>
```

Comment:
`hx` tags are supposed to be headings of sections, but here (and I've seen this elsewhere), it's being used stylistically (`h4` is about the right size that you want).

We might want to make this the correct tag semantically (e.g. in a `p` element instead, but a `div` usually works just fine). Elsewhere you've used the `Text__headerScale` mixin which I think is a good way to achieve your desired convenience while also keeping things semantic.

You can also make a `Text--h4` modifier if you're really after some quick-to-hand convenience.